src/cloud_common/protocols/unit/types.py

```diff
@@ -76,12 +77,13 @@
     str,
     Field(
         alias='nodeType',
-        title='Node type',
-        description='The type of the node.',
+        title='NodeType',
+        description='Depending on the configuration nodes are separated into several categories, named node types.',
         examples=[
-            'DomA',
-            'DomD',
-        ],
+            'VM setup has vm-dev-dynamic-1.0-main & vm-dev-dynamic-1.0-secondary node types:'
```

Comment:
This is a particular case that related to VM. Should not be part of generic description. Just few examples should be ok.

Reply:
removed